cubeviz/data_factories/__init__.py

```diff
@@ -229,6 +231,14 @@ def _any(self, conditionals):
     # Leaf processing
     #
 
+    def _has_data(self):
+        """
+        Used for miscellaneous cubes that we have not planned for by just making sure data in the cube exists. This
+        is currently only in use in the default.yaml file
+        :return:
+        """
+        return hasattr(self._fits[0], 'data')
```

Comment:
@javerbukh I think it might be safer to see if any HDU has an attribute data rather than just the first.  I think, too, that it might be better to see if it exists (as you have) AND that the len(self._fits[x].data) == 3. I think it is fair to say we are expecting a 3D cube of data.